sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -2,7 +2,7 @@
 
 [Azure TextAnalytics](https://azure.microsoft.com/services/cognitive-services/text-analytics/) is a cloud-based service that provides advanced natural language processing over raw text, and includes six main functions:
 
-__Note:__ This SDK targets Azure Text Analytics service API version 3.0.
+__Note:__ This SDK targets Azure Text Analytics service API version 5.1.0-preview.1.
```

Comment:
The service is still 3.x, even though our client SDK is version 5.x.

Reply:
my bad, will fix it.